app/goals/[id]/page.tsx

```
@@ -1,106 +1,55 @@
 'use client';
-
 import React, { useEffect } from 'react';
-
-import { setCookie } from 'cookies-next';
-
-import api from '@/api/clientActions/authApi';
+import { useRouter } from 'next/navigation';
+import Image from 'next/image';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)